docs/api/eth/tools/api.tools.fixtures.rst

```diff
@@ -60,129 +60,138 @@ sequence of functions.
-    | ``"gas"``          | the amount of gas allocated for the call                   |
-    +--------------------+------------------------------------------------------------+
-    | ``"code"``         | the bytecode to execute                                    |
-    +--------------------+------------------------------------------------------------+
-    | ``"vyperLLLCode"`` | the code in Vyper LLL (compiled to bytecode automatically) |
-    +--------------------+------------------------------------------------------------+
-
-
-.. autofunction:: eth.tools.fixtures.fillers.expect
-
-    This specifies the expected result of the test.
-
-    For state tests, multiple expectations can be given, differing in the transaction data, gas
-    limit, and value, in the applicable networks, and as a result also in the post state. VM tests
-    support only a single expectation with no specified network and no transaction (here, its role is
-    played by :func:`~eth.tools.fixtures.fillers.execution`).
-
-    * ``post_state`` is a list of state definition in the same form as expected
-      by :func:`~eth.tools.fixtures.fillers.pre_state`. State items that are
-      not set explicitly default to their pre state.
-
-    * ``networks`` defines the forks under which the expectation is applicable. It should be a sublist of
-      the following identifiers (also available in `ALL_FORKS`):
-
-      * ``"Frontier"``
-      * ``"Homestead"``
-      * ``"EIP150"``
-      * ``"EIP158"``
-      * ``"Byzantium"``
-
-    * ``transaction`` is a dictionary coming in two variants. For the main shard:
-
-      +----------------+-------------------------------+
-      | key            | description                   |
-      +================+===============================+
-      | ``"data"``     | the transaction data,         |
-      +----------------+-------------------------------+
-      | ``"gasLimit"`` | the transaction gas limit,    |
-      +----------------+-------------------------------+
-      | ``"gasPrice"`` | the gas price,                |
-      +----------------+-------------------------------+
-      | ``"nonce"``    | the transaction nonce,        |
-      +----------------+-------------------------------+
-      | ``"value"``    | the transaction value         |
-      +----------------+-------------------------------+
-
-    In addition, one should specify either the signature itself (via keys ``"v"``, ``"r"``, and ``"s"``) or
-    a private key used for signing (via ``"secretKey"``).
+    (address, "balance", <account balance>)
+    (address, "storage", <storage slot>, <storage value>)
+    (address, "storage", {<storage slot>: <storage value>})
+    (address, {"balance", <account balance>})
+
+.. automodule:: eth.tools.fixtures.fillers
+  :noindex:
+  :members: execution
+
+
+For VM tests, this function specifies the code that is being run as well as the current state of
```

Comment:
@pipermerriam I actually think we should move these docstrings *into* the function because as we do with the other API docs. That ensures they stay more up to date and is more in line with the rest of the API section which all has the docstrings right at the implementation.

Didn't want to change that in this PR though as I just wanted to get the live docs fixed without changing the visuals.

Reply:
:+1: I actually did that with the builder ones but forgot to apply it here.  Are you good doing that cleanup in a follow up PR?

Reply:
Sure thing :+1: